cluster/network.go

```diff
@@ -41,6 +41,9 @@ const (
 	KubeProxyPort       = "10256"
 	FlannetVXLANPortUDP = "8472"
 
+	ProtocolTCP = "TCP"
+	ProtocolUDP = "UDP"
```

Comment:
Is this used somewhere?

Reply:
Will be used later to support UDP checks.